lib/index.js

```diff
@@ -1,5 +1,5 @@
 var BroccoliSassCompiler = require('./broccoli_sass_compiler');
-var Eyeglass = require('eyeglass');
+var Eyeglass = require('eyeglass').Eyeglass;
```

Comment:
These are all early releases. I'm not especially worried about supporting older versions.